### DISCUSSION TITLE
Simplified options

### DISCUSSION THREAD

**Discussion by AntoninJcb** [abraunegg/onedrive#1446] [Ideas]
Hello!

Sorry if this has already been discussed, I couldn't find similar threads with the "option" keyword.

Anyway, have you thought about having simplified parameters? Like `onedrive -s` for --synchronize.

If often misspell parameters when I need to write them completely. It would be much faster to just type `onedrive -s` on the fly than the full "synchronize" word everytime.

What are your thoughts on this? I'm sure you have a good reason for not doing it, thank you for letting me know about it.

A. JACOB

**Comment by abraunegg** [comment]
@AntoninJcb 
Shorthand / simplified parameters are not included due to confusion with other binaries which have similar shorthand codes. 

Example, a user early on in the piece confused `-d` because of some other binary option, and could not understand why the client would never upload data ... at the time, `-d` stood for `--download-only` thus `-d` was removed.

This is something I may look at in the future again if there is enough support for it. Please open a feature request.
